### PR TITLE
Award supermentor badge

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- Add supermentor badge model
- Award supermentor badge when role is added

### CODE REVIEW

File: app/models/badges/supermentor_badge.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Badges::SupermentorBadge < Badge
+  seed 'Supermentor',
+    :legendary,
+    'supermentor',
+    'Mentored 100 students on a track'
```
Comment (ErikSchierboom) [inline app/models/badges/supermentor_badge.rb:5]: Do you want this to maybe also say something about the satisfaction rating?

Something like:

```suggestion
    'Satisfactorily mentored 100 students on a track'
```

**Review by ErikSchierboom — COMMENTED** [review]
This will also require a badge to be designed :)

File: app/models/badges/supermentor_badge.rb (hunk 1 above)
Comment (iHiD) [reply app/models/badges/supermentor_badge.rb:5]: I think I'm ok with just "Mentored"

**Review by iHiD — APPROVED** [review]
Nice :)

**Pull request merged**